libpdf/catalog.py

```diff
@@ -40,44 +41,51 @@ def get_named_destination(pdf):  # pylint: disable=too-many-branches
     name_tree = {}
     pdf_catalog = pdf.doc.catalog
     if 'Names' in pdf_catalog:
+        # PDF 1.2
         if isinstance(pdf_catalog['Names'], PDFObjRef) and 'Dests' in pdf_catalog['Names'].resolve():
             name_tree = pdf_catalog['Names'].resolve()['Dests'].resolve()
         elif isinstance(pdf_catalog['Names'], dict) and 'Dests' in pdf_catalog['Names']:
             name_tree = pdf_catalog['Names']['Dests'].resolve()
+        # check if name tree not empty
+        if not name_tree:
+            LOG.info('Catalog extraction: name destination exists but is empty...')
```

Comment:
Is this good or bad from a users' perspective?

Please remove the 3 dots. It either looks like the program processes something currently or something looks fishy in the PDF but the tool does not tell what.

Reply:
Done, removed.